templates/style.php

```diff
@@ -138,17 +138,17 @@
 
 /* Titlebar */
 nav.amp-wp-title-bar {
-	background: #0a89c0;
+	background: <?php echo get_theme_mod( 'amp_navbar_background' ); ?>;
 	padding: 0 16px;
 }
 
 nav.amp-wp-title-bar div {
 	line-height: 54px;
-	color: #fff;
+	color: <?php echo get_theme_mod( 'amp_navbar_color' ); ?>;
 }
 
 nav.amp-wp-title-bar a {
-	color: #fff;
+	color: <?php echo get_theme_mod( 'amp_navbar_color' ); ?>;
```

Comment:
We should probably pass these through template data instead (to allow for more easier filtering). Also, I'm not sure if `get_theme_mod` already sanitizes the values, but if not, we should probably escape them on output.